ritm_annotation/core/annotation/session.py

```
@@ -0,0 +1,327 @@
+        clicker = Clicker()
+        for click in obj.clicks:
+            # Convert from state Click (x, y) to inference Click (is_positive, coords=(y, x))
+            inference_click = InferenceClick(
+                is_positive=click.is_positive, coords=(click.y, click.x)
+            )
+            clicker.add_click(inference_click)
+
+        # Get prediction
+        try:
+            pred = self.predictor.get_prediction(clicker)
+            return pred
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)